src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -48,7 +48,7 @@
   @Getter(lazy = true)
   private static final String domain = getEnvType().getDomain();
 
-  private Configuration() {}
+  public Configuration() {}
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Hide this public constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1118)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1186)